mcclim.asd

```diff
@@ -66,10 +66,11 @@ interface management system."
 (defsystem #:mcclim/looks
   :depends-on (#:clim
                #-(or mcclim-beagle mcclim-ugly)
-               #:mcclim-clx/pretty              #| adds truetype        |#
-               #+mcclim-ugly #:mcclim-clx       #| raw clim-clx backend |#
-               #-mcclim-beagle #:mcclim-clx-fb  #| experimental backend |#
-               #+mcclim-beagle #:mcclim-beagle  #| OSX native (clozure) |#
+               #+old-fonts #:mcclim-clx/truetype  #| adds truetype        |#
+               #:mcclim-clx/freetype              #| adds freetype        |#
+               #+mcclim-ugly #:mcclim-clx         #| raw clim-clx backend |#
+               #-mcclim-beagle #:mcclim-clx-fb    #| experimental backend |#
```

Comment:
what is default for mcclim-clx-fb?

Reply:
I have not touched that library at all. The custom font rendering protocol is only part of mcclim-clx.